tldw_Server_API/app/api/v1/endpoints/voice_assistant.py

```
@@ -1457,6 +1457,98 @@ async def _stream_workflow_progress(
         )
 
 
+#######################################################################################################################
+#
+# Voice Command Dry-Run
+#
+
+class VoiceCommandDryRunRequest(BaseModel):
```

Ruff (0.15.7)

[error] Undefined name `BaseModel`

(F821)